reports/2022.html

```diff
@@ -1330,16 +1330,26 @@ <h3>Description</h3>
       <section>
         <h3>Status</h3>
         <ul>
-          <li>---</li>
+          <li><a href="https://chromestatus.com/feature/5711021289242624" target="_blank">Chrome</a> (Shipped)</li>
+          <li><a href="" target="_blank">Firefox</a> (Shipped)</li>
+          <li><a href="https://webkit.org/status/#feature-imperative-slot-api" target="_blank">Safari (Shipped?)</a></li>
         </ul>
       </section>
       <section>
         <h3>Initial API Summary/Quick API Proposal</h3>
-        <p>Summary or proposal based on current status; paragraph(s) and code.</p>
+        <p>Currently the Slot API only supports a declarative API, meaning that slot usage can only be expressed through adding the name attribute on an element.  But there are valid cases where from multiple sources of slotted content, the Web Components author may want to programmatically set the content of a slot instead.  Take <a href="https://github.com/WICG/webcomponents/blob/gh-pages/proposals/Imperative-Shadow-DOM-Distribution-API.md" target="_blank">this example</a> from the proposal.</p>
+        <pre class="html">
+          &lt;custom-tab show-panel="2"&gt;
+            &lt;tab-panel>&lt;/tab-panel&gt;
+            &lt;tab-panel>&lt;/tab-panel&gt;
+            &lt;tab-panel>&lt;/tab-panel&gt;
+          &lt;/custom-tab&gt;
+        </pre>
+        <p>Without an imperative API, how would an author be able to set the contents of a single available slot to the selected panel index?</p>
       </section>
       <section>
         <h3>Key Scenarios</h3>
-        <p>---</p>
+        <p>Some of the scenarios called out in the proposal include not having to pre-compute the slot names ahead of time, as well as being able to conditionally load content into a slot.</p>
```

Comment:
Would be cool if @cdata might be able to share whether this API does nice things for `<model-viewer>` dynamic slot situation…